src/java/org/apache/cassandra/service/StorageProxy.java

```diff
@@ -1027,11 +1011,6 @@ public static void mutateMV(ByteBuffer dataKey, Collection<Mutation> mutations,
                 // add a handler for each mutation - includes checking availability, but doesn't initiate any writes, yet
                 for (Mutation mutation : mutations)
                 {
-                    if (hasLocalMutation(mutation))
-                        writeMetrics.localRequests.mark();
-                    else
-                        writeMetrics.remoteRequests.mark();
-
```

Comment:
This codepath seems to be actuated by writes on the MV's base table, so it didn't seem correct to record anything around these metrics.